src/encoded/static/components/item-pages/ExperimentSetView.js

```diff
@@ -442,7 +476,8 @@ export class ProcessedFilesStackedTableSection extends React.PureComponent {
         };
 
         if (currentlyVisualizedFiles && currentlyVisualizedFiles.length > 0){
```

Comment:
I think we now no longer need to test for `currentlyVisualizedFiles` (nor `files` further downstream in HiGlassAdustableRow etc.) since only presence of higlassItem(s) should matter now (?)

We could put `const higlassItem = ProcessedFilesStackedTableSection.getHiglassItemFromProcessedFiles(context);` up above in place of `const currentlyVisualizedFiles = ProcessedFilesStackedTableSection.findAllFilesToVisualize(context);` and then change from `if (currentlyVisualizedFiles && currentlyVisualizedFiles.length > 0){` to `if (higlassItem && object.itemUtil.atId(higlassItem)){`.

Then we can stop passing in `files` in `hiGlassProps` object, and change `if (!files || !mounted || !higlassItem) {` to `if (!mounted || !higlassItem || object.itemUtil.atId(higlassItem)) {`. (The extra atId check would be just to doublecheck that User has permissions to view it as well, just in case they don't have view permission for whatever reason).

Reply:
I don't want to remove files from HiGlassAdjustableWidthRow yet, because the Other Processed Files section still relies on it for the collapsedToolTipContent. But I'll remove it from ProcessedFilesStackedTableSection.

Reply:
ProcessedFilesStackedTable uses currentlyVisualizedFiles to display the file table, so I'll keep that. But HiGlassAdustableRow no longer needs it.